package.json

```diff
@@ -11,31 +11,30 @@
     "vow-fs": "^0.3.4"
   },
   "devDependencies": {
-    "bem-components": "bem/bem-components#v2.1.0",
```

Comment:
With `bower` it's still possible but at least `bower` will give you notice that there's something wrong.

`npm` is about JS modules which has scoping. `bower` is for client side code with CSS. why not to use specific tools for specific tasks? that's recomended way in our docs and in `islands` library inside Yandex.